CHANGELOG.md

```diff
@@ -3,19 +3,61 @@ Release built: _not released yet_
 
 > [!CAUTION]
 > **Breaking Changes:**
-> - Changed ordering of entity metadata returned from `/state/entity/metadata`, `/state/entity/page/metadata` endpoints. Entries are no longer ordered by their last modification state version but rather by their first appearance on the network, descending.
+> - Changed ordering of entity metadata. Entries are no longer ordered by their last modification state version but rather by their first appearance on the network, descending. Affected endpoints:
+>   - `/state/entity/metadata`
+>   - `/state/entity/page/metadata`
+> - Changed ordering of fungible and non fungible resources. Entries are no longer ordered by their last modification state version but rather by their first appearance on the network, descending. Affected endpoints:
+>   - `/state/entity/details`
+>   - `/state/entity/page/fungibles/`
+>   - `/state/entity/page/non-fungibles/`
+> - Changed ordering of vaults when using vault aggregation level. Entries are no longer ordered by their last modification state version but rather by their first appearance on the network, descending.  Affected endpoints:
+>   - `/state/entity/details`
+>   - `/state/entity/page/fungibles/`
+>   - `/state/entity/page/fungible-vaults/`
+>   - `/state/entity/page/non-fungibles/`
+>   - `/state/entity/page/non-fungible-vaults/`
+> - Changed ordering of non fungible ids. Entries are no longer ordered by their last modification state version but rather by their first appearance on the network, descending. Affected endpoints:
+>   - `/state/entity/page/non-fungible-vault/ids`
+>   - `/state/entity/details` (when using `non_fungible_include_nfids` opt-in)
+>   - `/state/entity/page/non-fungibles/` (when using `non_fungible_include_nfids` opt-in)
+>   - `/state/entity/page/non-fungible-vaults/` (when using `non_fungible_include_nfids` opt-in)
+> -  Existing non fungible vaults with no items will no longer return `items: null` and will return an empty array `items: []` instead, as we do in all other collections. Affected endpoints:
+>   - `/state/entity/page/non-fungible-vault/ids`
+>   - `/state/entity/details` (when using `non_fungible_include_nfids` opt-in)
+>   - `/state/entity/page/non-fungibles/` (when using `non_fungible_include_nfids` opt-in)
+>   - `/state/entity/page/non-fungible-vaults/` (when using `non_fungible_include_nfids` opt-in)
 
 
 ### Bug fixes
+- Fixed `total_count` property returned for `explicit_metadata` collection. It returns now total number of metadata items for given entity. Affected endpoints:
```

Comment:
I'm not sure this necessarily is a bug fix? Perhaps let's discuss

Reply:
previously we returned count of explicit metadata items returned I'd consider that a bug.

Reply:
Well in the context of the collection under consideration being "the explicit list of keys to lookup" the collection size is the count of metadata items returned. But I'm happy to change it.

If we want to change it, we should clarify here and in the schema if it's the `total number of undeleted metadata items` or `total number of metadata items that have existed against this entity`